src/custom/pages/Faq/index.tsx

```diff
@@ -253,6 +253,22 @@ export default function Faq() {
         <Content>
           <h2 id="protocol">Protocol</h2>
 
+          <h3 id="does-cowswap-have-a-token">Does CowSwap have a token?</h3>
+
+          <p>
+            There is currently no plan for a CowSwap specific token. At the moment the value is already captured by the
+            GNO token, as CowSwap is built on top of Gnosis Protocol. If you are curious about how the value is
+            captured, you can <a href="https://forum.gnosis.io/t/gpv2-fee-model/1266"> read this forum post</a>.
+          </p>
+
+          <p>
+            Be cautious, the{' '}
+            <a href="https://etherscan.io/token/0xBF04E48C5d8880306591ef888cde201D3984Eb3E">CowSwap token</a> listed on
+            Uniswap is not affiliated with this project in any way, shape or form. Please note that this is{' '}
+            <strong>not</strong>
```

Comment:
![image](https://user-images.githubusercontent.com/2352112/118719450-cb10a380-b828-11eb-8694-b0fcf55bd8b2.png)


```suggestion
            <strong>not</strong>&nbsp;
```

Reply:
@anxolin like the reformulation. But since Its for the specific COW token, I would leave like this: 

Be cautious, some people may create fake COW tokens, that are not affiliated with this project. Please note that any token listed in any AMM is not associated with this project in any way, shape or form.